### PR TITLE
P17a & P17b recovered ingestion csvs

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP05MOAS-GL380/CP05MOAS-GL380_R00011_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/merged/cp_*.mrg,CP05MOAS-GL380-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/dvl/*.PD0,CP05MOAS-GL380-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/merged/cp_*.mrg,CP05MOAS-GL380-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/merged/cp_*.mrg,CP05MOAS-GL380-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/merged/cp_*.mrg,CP05MOAS-GL380-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00011/merged/cp_*.mrg,CP05MOAS-GL380-05-PARADM000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL380/CP05MOAS-GL380_R00011_ingest.csv:7]: looks good - data available.

File: CP05MOAS-GL388/CP05MOAS-GL388_R00012_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/merged/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/dvl/*.PD0,CP05MOAS-GL388-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/merged/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/merged/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/merged/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00012/merged/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,recovered_host,Available,
```
Comment (rtravis-whoi) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00012_ingest.csv:7]: looks good - data available

**Review by rtravis-whoi — APPROVED** [review]

File: CP05MOAS-GL380/CP05MOAS-GL380_R00011_ingest.csv (hunk 1 above)
Comment (snwhite) [reply CP05MOAS-GL380/CP05MOAS-GL380_R00011_ingest.csv:7]: Looks good.

File: CP05MOAS-GL388/CP05MOAS-GL388_R00012_ingest.csv (hunk 2 above)
Comment (snwhite) [reply CP05MOAS-GL388/CP05MOAS-GL388_R00012_ingest.csv:7]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**